hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/scm/AbstractReconContainerManagerTest.java

```diff
@@ -129,4 +132,46 @@ private StorageContainerServiceProvider getScmServiceProvider()
         .thenReturn(containerWithPipeline);
     return scmServiceProviderMock;
   }
+
+  protected Table<ContainerID, ContainerInfo> getContainerTable()
+      throws IOException {
+    return CONTAINERS.getTable(store);
+  }
+
+  protected ContainerWithPipeline getTestContainer(LifeCycleState state)
+      throws IOException {
+    ContainerID containerID = new ContainerID(100L);
+    Pipeline pipeline = getRandomPipeline();
```

Comment:
Can we reuse the getTestContainer below to reduce duplicated code?

Reply:
Sure, will do.